lib/compiler.js

```diff
@@ -1,9 +1,13 @@
 var hasOwn = Object.prototype.hasOwnProperty;
 var assert = require("assert");
-var types = require("ast-types");
+var types = require("ast-types/fork")([
+  require("ast-types/def/esprima"),
+  require("ast-types/def/babel6")
+]);
+
 var n = types.namedTypes;
 var b = types.builders;
-var MagicString = require("magic-string");
+var MagicString = require("magic-string/dist/magic-string.cjs.js");
 var utils = require("./utils.js");
```

Comment:
I noticed that by default `magic-string/dist/magic-string.es.js` is loaded. I think that's because by this time the loader is already registered and so the es flavor is loaded.